src/validatedODEs.jl

```diff
@@ -215,19 +215,20 @@ function validated_step!(f!, t::Taylor1{T},
     # If after 25 checks `check_property` is not satisfied, thow an error.
     nsteps += 1
     issatisfied = false
-    Δ = zero.(δq_norm)
+    # Δ = zero.(δq_norm)
     for nchecks = 1:25
         # Validate the solution: remainder consistent with Schauder thm
         Δ = remainder_taylorstep!(f!, t, x, dx, xI, dxI, δq_norm, Interval(0.0, δt))
+        rem .= rem .+ Δ
 
         # Create TaylorModelN to store remainders and evaluation
         @inbounds begin
             for i in eachindex(x)
-                auxTM = fp_rpa( TaylorModelN(x[i](0..δt), rem[i]+Δ[i], q0, q0box) )
-                xTMN[i] = absorb_remainder(auxTM)
-                rem[i] = remainder(xTMN[i])
+                xTMN[i] = fp_rpa( TaylorModelN(x[i](0..δt), rem[i], q0, q0box) )
+
                 # If remainder is still too big, do it again
-                if mag(rem[i]) > 1.0e-10
+                j = 0
+                while (j < 10) && (mag(rem[i]) > 1.0e-10)
```

Comment:
was the intention to increase `j` in the while loop or something?

Reply:
Thanks! Good catch! I'll push a corrected version.